zh_CN/readme.md

```diff
@@ -0,0 +1,955 @@
+
+```js
+test.beforeEach(t => {
+    t.context.data = generateUniqueData();
+});
+
+test(t => {
+    t.is(t.context.data + 'bar', 'foobar');
+});
+```
+
+默认情况下`t.context`是一个对象，但你可以重新赋值：
+
+```js
+test.beforeEach(t => {
+    t.context = 'unicorn';
+});
+
+test(t => {
+    t.is(t.context, 'unicorn');
+});
+```
+
+上下文共享在`before`和`after`钩子中*不*可用。
+
+### 连接测试修饰符
+
+你可以以任何顺序使用`.serial`，`.only`和`.skip`，与`test`，`before`，`after`，`beforeEach`和`afterEach`一起，比如：
+
+```js
+test.before.skip(...);
+test.skip.after(...);
+test.serial.only(...);
+test.only.serial(...);
+```
+
+这意味着你可以在每个测试或钩子的末尾临时添加`.skip`或`.only`，而不用做其他的修改。
+
+### 自定义断言
+
+你可以使用其他断言库来替代内置的断言库，当断言失败可以让其抛出异常。
+
+但这样做的话你将得不到[内置断言库](#断言)的良好体验，同时你也将不会用到[断言计划](#断言计划) ([看#25](https://github.com/sindresorhus/ava/issues/25))。
+
+```js
+import assert from 'assert';
+
+test(t => {
+    assert(true);
+});
+```
+
+### 支持ES2015
+
+AVA 通过[Babel 6](https://babeljs.io)内置支持ES2015，只需要用ES2015的方式写你测试，不需要额外的配置。你可以在你的工程中使用任何的Babel 版本，我们使用我们捆绑的Babel，带有[`es2015`](https://babeljs.io/docs/plugins/preset-es2015/)和[`stage-2`](https://babeljs.io/docs/plugins/preset-stage-2/)设置，和[`espower`](https://github.com/power-assert-js/babel-plugin-espower)和[`transform-runtime`](https://babeljs.io/docs/plugins/transform-runtime/) 插件一样。
+
+类似的Babel 配置同样适用于AVA，如下：
+
+```json
+{
+  "presets": [
+    "es2015",
+    "stage-2",
+  ],
+  "plugins": [
+    "espower",
+    "transform-runtime"
+  ]
+}
+```
+
+你可以自定义AVA 如何通过[`package.json`配置](#配置)`babel`选项来转换测试文件，例如你可以这样来覆盖配置：
+
+```json
+{
+  "ava": {
+    "babel": {
+      "presets": [
+        "es2015",
+        "stage-0",
+        "react"
+      ]
+    }
+  },
+}
+```
+
+你也可以使用特殊的`"inherit"`关键字，这让AVA 遵从你的[`.babelrc`或`package.json`文件](https://babeljs.io/docs/usage/babelrc/)中的Babel 配置，这种方式让测试文件的转换方式和与源文件的相同，可以无需在AVA 中重复配置。
+
+ ```json
+{
+    "babel": {
+        "presets": [
+            "es2015",
+            "stage-0",
+            "react"
+        ]
+    },
+    "ava": {
+        "babel": "inherit",
+    },
+}
+```
+
+注意，AVA*总是*应用[`espower`](https://github.com/power-assert-js/babel-plugin-espower)和[`transform-runtime`](https://babeljs.io/docs/plugins/transform-runtime/) 插件。
+
+### 支持TypeScript
+
+AVA 支持TypeScript，你必须自己配置转换规则，当你在你的`tsconfig.json`文件中把`module`设为`commonjs`，TypeScipt 将自动为AVA 找到类型定义。你应该把`target`设置为`es2015`来使用promises 和async 函数。
+
+### 转换导入模块
+
+AVA 现在只转换需要运行的测试，*它不会转换那些在测试中你`import`的模块*，这可能不是你所期望的但这就是现在的工作方案。
+
+如果你使用Babel 你可以使用它的[require 钩子](https://babeljs.io/docs/usage/require/)来实时转换导入模块，运行AVA 带上`--require babel-register` (请看[CLI](#cli))参数或[在你的`package.json`里配置](#配置)。
+
+你也可以在另外一个进程里转换你的模块，并且参考你的转换文件而不是你的测试源码。
+
+### 支持Promise
+
+如果你在测试里返回一个promise，你不需要在测试里明确的结束测试，当promise resolve 的时候它会自己结束。
+
+```js
+test(t => {
+    return somePromise().then(result => {
+        t.is(result, 'unicorn');
+    });
+});
+```
+
+### 支持Generator
+
+AVA 自带对[generator 函数](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)的内置支持。
```

Comment:
Replace:
`https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*`
by this chinese link is better:
`https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Statements/function*`